.github/workflows/check-vertexai-responses.yml

```diff
@@ -5,6 +5,7 @@ on: pull_request
 jobs:
   check-version:
     runs-on: ubuntu-latest
+    permissions: write-all
```

Comment:
should it be  `pull-requests: write` ?

Reply:
Yep, just wanted to make sure the write perms were needed at all, it's still succeeding with this change